installers/gcp-marketplace/application.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: app.k8s.io/v1beta1
+kind: Application
+metadata:
+  name: '${OPERATOR_NAME}'
+  labels:
+    app.kubernetes.io/name: '${OPERATOR_NAME}'
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: '${OPERATOR_NAME}'
+  componentKinds:
+    - { group: core, kind: ConfigMap }
+    - { group: core, kind: Secret }
+    - { group: core, kind: Service }
+    - { group: apps, kind: Deployment }
+    - { group: batch, kind: Job }
+  descriptor:
+    description: Enterprise PostgreSQL-as-a-Service for Kubernetes
+    type: Crunchy PostgreSQL Operator
+    version: '${PGO_VERSION}'
+    icons:
+      - src: https://raw.githubusercontent.com/CrunchyData/crunchy-containers/master/images/crunchy_logo.png
+    maintainers:
+      - name: Crunchy Data
+        url: https://www.crunchydata.com/
+        email: info@crunchydata.com
+    keywords:
+      - postgres
+      - postgresql
+      - database
+      - sql
+      - operator
+      - crunchy data
+    links:
+      - description: GitHub
+        url: https://github.com/CrunchyData/postgres-operator
```

Comment:
This link appears with the text "Crunchy PostgreSQL Operator" in the Console's Application listing. Perhaps because it is the first in this list?

Reply:
Will get clarification on what link we want to be first.